src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random;
+    private final ColorSupplier colorSupplier;
+
+    public Lottery() {
+        random = new Random();
+        colorSupplier = new ColorSupplier();
+    }
+
+    public Ball getRandomBall() {
+        final int max = 100;
+        return new Ball(random.nextInt(max), colorSupplier.getRandomColor());

```

Comment:
This comment is ignored 'Also create variables for rand.nextInt(100) and colorSupplier.getRandomColor(). This makes the code more readable'